<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CompenInsertDialog</class>
 <widget class="QDialog" name="CompenInsertDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>949</width>
    <height>477</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>理赔业务编号:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>320</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>汽车轮胎:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>140</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>车内赔付值:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_EngineEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>250</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_NumberEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>120</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>220</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>车窗赔付值:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_CaseNumberEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>220</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Commit">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>390</y>
     <width>171</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>提交</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_AllEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>170</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>20</y>
     <width>861</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>20</pointsize>
    </font>
   </property>
   <property name="text">
    <string>定损登记</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>180</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>实际总共赔付值:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>270</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>车身赔付(元):</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_CarNumberEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>170</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>310</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>描述:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>220</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>案件编号:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_PayEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>270</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>170</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>车牌号：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Reset">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>重置</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Insert_Return">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>390</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>返回</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_SingleEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>130</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_WheelEdit">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>320</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>260</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>汽车发动机赔付值:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Insert_WindowEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>210</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="Insert_DescriptionEdit">
   <property name="geometry">
    <rect>
     <x>670</x>
     <y>290</y>
     <width>191</width>
     <height>85</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>431</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>80</y>
     <width>431</width>
     <height>361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>440</y>
     <width>861</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_6</zorder>
  <zorder>label_13</zorder>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>label_11</zorder>
  <zorder>Insert_EngineEdit</zorder>
  <zorder>Insert_NumberEdit</zorder>
  <zorder>label_7</zorder>
  <zorder>Insert_CaseNumberEdit</zorder>
  <zorder>Insert_Commit</zorder>
  <zorder>Insert_AllEdit</zorder>
  <zorder>label_10</zorder>
  <zorder>label_12</zorder>
  <zorder>label_5</zorder>
  <zorder>Insert_CarNumberEdit</zorder>
  <zorder>label_9</zorder>
  <zorder>label_4</zorder>
  <zorder>Insert_PayEdit</zorder>
  <zorder>label_3</zorder>
  <zorder>Insert_Reset</zorder>
  <zorder>Insert_Return</zorder>
  <zorder>Insert_SingleEdit</zorder>
  <zorder>Insert_WheelEdit</zorder>
  <zorder>label_8</zorder>
  <zorder>Insert_WindowEdit</zorder>
  <zorder>Insert_DescriptionEdit</zorder>
  <zorder>label_14</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
